lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -711,7 +711,7 @@ export abstract class CacheManager implements ICacheManager {
      * @param scopes
      * @param authScheme
      */
-    readAccessTokenFromCache(clientId: string, account: AccountInfo, request: BaseAuthRequest): AccessTokenEntity | null {
+    readAccessTokenFromCache(clientId: string, account: AccountInfo, request: BaseAuthRequest, hash?: string): AccessTokenEntity | null {
```

Comment:
I think we now have a few different hashes involved, can we name this something more specific?

Reply:
Ideally this can be avoided I guess, as we have a separate look up added in the OBOClient. We need to have this only in the filter code. I had hopes to reuse the existing look-up but it adds a lot of refactoring which I am not inclined to make at this point. Let me see if I can remove this here.

Reply:
Removed this, not necessary for this flow.